src/usr/local/www/system_groupmanager.php

```diff
@@ -254,7 +254,7 @@ function build_priv_table() {
 	if ($user_has_root_priv) {
 		$privhtml .=		'<tr>';
 		$privhtml .=			'<td colspan="2">';
-		$privhtml .=				'<b>' . gettext('Security notice: Users in this group effectively have root privilege') . '</b>';
+		$privhtml .=				'<b>' . gettext('Security notice: Users in this group effectively have administrator-level privilege') . '</b>';
```

Comment:
Should be plural ("privileges") with the new wording, or "access" (see later comments)